Update the onboarding flow to replace the Privacy Policy link with the Data Processing Agreement (#27934)
directus/directus: .changeset/onboarding-dpa-replace-privacy.md

---
'@directus/app': patch
'@directus/constants': patch
---

> Updated the onboarding flow to replace the Privacy Policy link with the Data Processing Agreement.

Reviewer: ```suggestion
Updated the onboarding flow to replace the Privacy Policy link with the Data Processing Agreement
```

> Replaced the Privacy Policy link with the Data Processing Agreement in the setup/onboarding license acceptance

Reviewer: ```suggestion
Updated the onboarding flow to replace the Privacy Policy link with the Data Processing Agreement.
```
